### PR TITLE
USSD Public functionality - User identification

### PR DESCRIPTION
**Pull request by gsvr**
Reach main menu state

### CODE REVIEW

File: test/ussd_public.test.js [hunk 1]
```diff
@@ -271,7 +288,7 @@ describe("Hello Mama app", function() {
                 });
             });
 
-            describe("Change states flows - baby messages", function() {
+            describe.skip("Change states flows - baby messages", function() {
```
Comment (gsvr) [inline test/ussd_public.test.js:318]: `.skip`s were left in on purpose to facilitate breaking up into different issues.

**Comment by gsvr** [comment]
@imsickofmaps Please review.

File: go-ussd_public.js [hunk 2]
```diff
@@ -1260,17 +1238,39 @@ go.app = function() {
...
+                        if (contact.details.household_msgs_only) {
+                            self.im.user.set_answer('receiver_household_only', true);
+                            return self.states.create('state_main_menu_household');
+                        } else {
+                            return self.states.create('state_main_menu');
+                        }
+                    });
+            }
+
+            // return go.utils_project
```
Comment (imsickofmaps) [inline go-ussd_public.js]: Needed?

File: test/ussd_public.test.js [hunk 3]
```diff
@@ -111,11 +109,22 @@ describe("Hello Mama app", function() {
...
                             ].join('\n')
                         })
+                        .check(function(api) {
+                            var expected_used = [0, 1];
+                            var fixts = api.http.fixtures.fixtures;
+                            // var i = 0;
+                            var fixts_used = [];
+                            fixts.forEach(function(f, i) {
+                                f.uses > 0 ? fixts_used.push(i) : null;
+                                // i++;
```
Comment (imsickofmaps) [inline test/ussd_public.test.js]: Delete

**Comment by imsickofmaps** [comment]
Two little comment things then :+1:

**Pull request merged**